nixos/tests/podman.nix

```diff
@@ -61,6 +61,12 @@ import ./make-test-python.nix (
           podman.succeed("podman stop sleeping")
           podman.succeed("podman rm sleeping")
 
+
+      podman.succeed(
+          "mkdir -p /tmp/podman-run-1000/libpod && chown alice -R /tmp/podman-run-1000"
+      )
```

Comment:
Thank you @adisbladis! :pray: I'm wondering if this is a Podman regression and there is a missing `mkdirAll` somewhere in the code base.

Reply:
I haven't debugged this in detail, but an educated guess is that it stems from us not executing commands from the context of a proper systemd user session where these directories would have been created at login time.

Reply:
Makes sense.